core/evancore.vue.libs/src/components/dapp-wrapper/dapp-wrapper.ts

```diff
@@ -315,6 +315,11 @@ export default class DAppWrapperComponent extends mixins(EvanComponent) {
       this.enableSidebar = false;
     }
 
+    // create fullPath for current routes to get correct active state
+    [...(this.routes || []), ...(this.bottomRoutes || [])].forEach((route) => {
```

Comment:
Why is the `|| []` necessary? Would be cleaner to make sure the arrays arent undefined beforehand. Just initialize them as `routes = []`?

Reply:
Routes are not needed, when the `enableSidebar` is false, so at this point i just added the empty array for error less destructuring.